pytools/mpiwrap.py

```diff
@@ -13,7 +13,8 @@
 pytools.prefork.enable_prefork()
 
 
-if Is_initialized():  # noqa pylint:disable=undefined-variable
+# pylint:disable=undefined-variable
```

Comment:
I *think* this disables that warning for the whole file. Is that the intention?

Reply:
My reading of their documentation
https://pylint.pycqa.org/en/latest/user_guide/messages/message_control.html#block-disables
is that if there's a statement on the next line, it disables it for that, and if there's not it disables it for the whole block.

I can put it back on the same line, it just looked a bit silly with 3 types of ignores :(

Reply:
Did you mean `disable-next`?

Reply:
Yep :facepalm: Somehow I assumed that it just worked™ with the normal disable..